regexp_test.go

```diff
@@ -576,17 +576,11 @@ func TestRegexpMatchAll(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if regex.self.(*regexpObject).getLastIndex() != 0 {
```

Comment:
This change broke the build...

Reply:
sorry, I woke up with the idea that I needed to drop it as it is in the JS code ... and then apparently never ran `go test`